src/components/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface HeaderProps {
+  todos: Todo[];
+  completedTodos: Todo[];
+  onTodosToggle: () => void;
+}
+
+export const Header: React.FC<HeaderProps> = ({
+  todos,
+  completedTodos,
+  onTodosToggle,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {todos.length > 0 && (
```

Comment:
change it in all the cases
```suggestion
      {...... && (
```